File: ILove.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    int num;
    cin >> num;
    int arr[num];
    for (int i=0; i<num; i++) 
        cin >> arr[i];
    int minNum = arr[0], maxNum = arr[0];
    int counter = 0;
    for (int i=1; i<num; i++) {
        if (arr[i] > maxNum) {
            maxNum = arr[i];
            counter++;
        }
        else if (arr[i] < minNum) {
            minNum = arr[i];
            counter++;
        }
    }
    cout << counter << endl;

    return 0;
}

File: Python/ILove.py
def main():
    num = int(input())
    arr = [int(input()) for i in range(num)]
    minNum = arr[0]
    maxNum = arr[0]
    counter = 0
    for i in range(1, num):
        if arr[i] > maxNum:
            maxNum = arr[i]
            counter += 1
        elif arr[i] < minNum:
            minNum = arr[i]
            counter += 1
    print(counter)


if __name__ == "__main__":
    main()
